utilities/manual/source/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # CHANGELOG
 ## TeXstudio 4.5.2
 
+- allow wheel usage over linenumber/linemark/fold panel of editors
 - fix skipping scroll when pointer was moved outside the text pane ([#2614](https://github.com/texstudio-org/texstudio/issues/2614))
+- add Min/Max/Close buttons to Macro Editor dialog ([2987](https://github.com/texstudio-org/texstudio/pull/2987))
+- status bar of Macro Editor behaves weird ([3001](https://github.com/texstudio-org/texstudio/pull/3001))
```

Comment:
the changelog should rather say what was changed, not whatever the title of a issue report was.

Reply:
which of the lines 5, 6, 7?

Reply:
"status bar of Macro Editor behaves weird"

Reply:
changed text